packages/platform-ios/src/commands/runIOS/__tests__/parseIOSDevicesList.test.ts

```diff
@@ -29,14 +29,29 @@ describe('parseIOSDevicesList', () => {
     );
 
     expect(devices).toEqual([
-      {
-        name: 'Maxs MacBook Pro',
-        udid: '11111111-1111-1111-1111-111111111111',
```

Comment:
why was this removed? we support running on host Mac for Catalyst

Reply:
Ohhh, that’s good to know, thanks! I guess the function name made me think it was pure iOS.

Reply:
I get you, we just recently added this :D